sample_site/_config.yml

```diff
@@ -15,3 +13,4 @@ asciidoctor:
     - idseparator=_
     - source-highlighter=coderay
     - icons=font
+theme: minima
```

Comment:
I couldn't see the difference with this theme.

Reply:
https://github.com/helaili/jekyll-action/pull/17/commits/379623cee30d46687dd2a138310ca7fb8a3dec1d
looks like I forgot to cleanup woops... can just revert that commit, I'll do it later.